tests/yam/verify_consistency_self_update.pm

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 SUSE LLC
+#
+# Summary: update the system using the specified repo and then
+#          open and close YaST2 control center using both Qt and Ncurses interfaces.
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use YaST::Module;
+
+sub run {
+    select_console 'root-console';
+
+    zypper_call 'ar -f ' . get_var('SELF_UPDATE_REPO') . ' self-update-repo';
+    zypper_call 'dup';
+    zypper_call 'lr --uri';
+
+    YaST::Module::open(module => 'system_settings', ui => 'ncurses');
```

Comment:
we might need to modify the library, what happen if you don't pass a module? in the command line it will open the YaST Control center, we should not need to open any other modules but this basic one, that would give us additional error unrelated with the self update.

Reply:
from the lib:


```
    die 'No module name specified.' unless defined $module;
    die 'No user interface specified.' unless defined $ui;
```